app/views/suppress_collections/index.html.erb

```diff
@@ -58,7 +58,7 @@
     <% @top_10_sources.each do |source| %>
     <tr>
       <td>
-        <strong><%= source['name'] %></strong> was activated by <strong><%= source['status_updated_by'] %></strong> at <strong><%= source['status_updated_at'].try(:to_datetime).try(:strftime, '%m/%d/%Y %I:%M%p') %></strong></td>
+        <strong><%= source['name'] %></strong> was activated by <strong><%= source['status_updated_by'] %></strong> at <strong><%= source['status_updated_at'].try(:to_datetime).try(:strftime, '%d/%m/%Y %I:%M%p') %></strong></td>
```

Comment:
The above would result in only one code change. It is something that can be improved over time rather then fixing all of the dates in one go.